groups/builtinroles.go

```diff
@@ -0,0 +1,7 @@
+package groups
+
+import "github.com/absmach/magistrala/pkg/roles"
+
+const (
+	BuiltInRoleAdmin roles.BuiltInRoleName = "admin"
+)
```

Comment:
Inline `const Admin roles.BuiltInRoleName = "admin"` (also, Admin name is enough - there is no need to to put type in the constant name).